Makefile

```diff
@@ -2,8 +2,7 @@ SHELL=/usr/bin/env bash -o pipefail
 
 # IMAGE_TAG_BASE defines the registry/namespace and part of the image name
 # This variable is used to construct full image tags for bundle and catalog images.
-#
-IMAGE_TAG_BASE ?= quay.io/sthaha/monitoring-stack-operator
+IMAGE_TAG_BASE ?= monitoring-stack-operator
```

Comment:
Isn't this `IMAGE_BASE` since there are no tags in the value?

Reply:
I am happy to change it :-)